modules/op-starting-pipelines.adoc

```diff
@@ -3,66 +3,68 @@
-* For the *Authentication Type* `Image Registry Crendentials`, specify the *Registry Server Address* that you want to authenticate, and provide your credentials in the *Username*, *Password*, and *Email* fields.
+* For the *Authentication Type* `Image Registry Credentials`, specify the *Registry Server Address* that you want to authenticate, and provide your credentials in the *Username*, *Password*, and *Email* fields.
 +
 Select *Add Credentials* if you want to specify an additional *Registry Server Address*.
 
 * For the *Authentication Type* `Basic Authentication`, specify the values for the *UserName* and *Password or Token* fields.
-* For the *Authentication Type* `SSH Keys`, specify the value for the *SSH Private Key* field.
+* For the *Authentication Type* `SSH Keys`, specify the value of the *SSH Private Key* field.
 ... Select the check mark to add the secret.
 
 +
-You can add multiple secrets based upon the number of resources in your Pipeline.
+You can add multiple secrets based upon the number of resources in your pipeline.
 
-. Click *Start* to start the PipelineRun.
-. The *Pipeline Run Details* page displays the Pipeline being executed. After the Pipeline starts, the Tasks and Steps within each Task are executed.
+. Click *Start* to start the pipeline.
+. The *Pipeline Run Details* page displays the pipeline being executed. After the pipeline starts, the tasks and steps within each task are executed.
 You can:
-
-* Hover over the Tasks to see the time taken for the execution of each Step.
-* Click on a Task to see logs for each of the Steps in the Task.
-* Click the *Logs* tab to see the logs according to the execution sequence of the Tasks and use the *Download* button to download the logs to a text file.
-+
-.Pipeline run
-image::op_pipeline_run.png[]
+* Hover over the tasks to see the time taken to execute each step.
+* Click on a task to see the logs for each step in the task.
+* Click the *Logs* tab to see the logs relating to the execution sequence of the tasks. You can also expand the pane and download the logs individually or in bulk, by using the relevant button.
+* Click the *Events* tab to see the stream of events generated by a pipeline run.
 +
+You can use the *Task Runs*, *Logs*, and *Events* tabs to assist in debugging a failed pipeline run or a failed task run.
+//+
+//.Pipeline run details
+//image::op_pipeline_run.png[Pipeline run details]
+//+
 //Add workspace and credential steps in this section.
 //Probably need a sep section for logging and monitoring pipelines.
-. For Pipelines created using the *From Git* option, you can use the *Topology* view to interact with Pipelines after you start them:
+. For pipelines created using the *From Git* option, you can use the *Topology* view to interact with pipelines after you start them:
 +
 [NOTE]
 ====
 To see Pipelines created using the *Pipeline Builder* in the *Topology* view, customize the Pipeline labels to link the Pipeline with the application workload.
```

Comment:
Most of the instances of "Pipelines" or "Pipeline" in this not should be in lower case for consistency with the rest of the assembly. You could also change the first "Pipelines" to "the pipelines". For example:

"To see the pipelines created using the *Pipeline Builder* in the *Topology* view, customize the pipeline labels to link the pipeline with the application workload."